src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -93,30 +98,30 @@ def _create_pool(self):
         # Create a pool
         self.pool.create()
 
-    def _create_cont(self):
-        """Create a TestContainer object to be used to create container."""
-        # TO-DO: Enable container using TestContainer object,
-        # once DAOS-3355 is resolved.
-        # Get Container params
-        #self.container = TestContainer(self.pool)
-        #self.container.get_params(self)
-
-        # create container
-        # self.container.create()
-        env = Dfuse(self.hostlist_clients, self.tmp).get_default_env()
-        # command to create container of posix type
-        cmd = env + "daos cont create --pool={} --svc={} --type=POSIX".format(
-            self.mdtest_cmd.dfs_pool_uuid.value, self.mdtest_cmd.dfs_svcl.value)
-        try:
-            container = subprocess.Popen(cmd, stdout=subprocess.PIPE,
-                                         shell=True)
-            (output, err) = container.communicate()
-            self.log.info("Container created with UUID %s", output.split()[3])
+    def _create_cont(self, doas_cmd):
```

Comment:
This looks like common code with fio so could be shared but I'm prepared to approve this to move forward.

Reply:
I agree. I am only refraining from doing so bcoz I know Ravi is working on enabling aggregation on some of the tests. So I would like to leave it for him to decide.

Reply:
I've been working with Amanda to implement a common code approach that would be implemented in the ExecutableCommand class that would define a method that would enable calling an inherited class's 'helper' method - like `container_create()` for DaosCommand - and it would apply a regex to the `CmdResult.stdout` to get the usful information  - in this case the container UUID.